configs/recommended.js

```diff
@@ -0,0 +1,89 @@
+const typescriptRules = {
+  '@typescript-eslint/no-var-requires': 'off',
+  '@typescript-eslint/no-explicit-any': 'off',
+  '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
Just to understand, why is this off? I find it useful to spot things that can be removed

Reply:
I was having issues with the `no-unused-vars`, `@typescript-eslint/no-unused-vars` and `@typescript-eslint/no-unused-vars-experimental` rules, but it's fixed now :smile_cat: